src/app/pages/viewing/messages/messages.component.css

```diff
@@ -23,22 +32,95 @@
 
 .page {
   background-color: var(--accent);
+  box-shadow: 0 0.15em 0.25em 0.05em rgba(0, 0, 0, 0.175);
   color: white;
   height: 70vh;
-  padding-left: 6em;
-  padding-right: 6em;
+  padding: 3em 6em 3em 6em;
   display: flex;
-  justify-content: center;
-  align-items: center;
+  overflow-y: auto;
+}
+
+.carousel-header {
+
+}
+
+.carousel-front-page-header {
+
+}
+
+.card-header-text {
+  font-weight: bold;
+  margin-top: 0.2em;
+}
+
+.card-header {
+  display: flex;
+  flex-direction: row;
+}
+
+.card {
+  display: inline-block;
 }
 
 .message-card {
   background-color: var(--light-grey);
-  height: auto;
-  min-height: 40em;
-  padding: 3em 6em 3em 6em;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  margin-bottom: 2em;
+  /*box-shadow: 0 0.15em 0.25em 0.05em rgba(0, 0, 0, 0.175);*/
+  width: 100%;
+  word-break: break-word;
+  max-height: 30em;
+  overflow-y: auto;
+}
+
+.profile-pic {
+  background: var(--light-grey);
+  height: 1.8em;
+  width: 1.8em;
+  border-radius: 100%;
+}
+
+@media (min-width: 576px) {
+  .card-columns {
+    column-count: 1;
+  }
+}
+
+@media (min-width: 768px) {
+  .card-columns {
+    column-count: 2;
+  }
+}
+
+@media (min-width: 992px) {
+  .card-columns {
+    column-count: 2;
+  }
+}
+
+@media (min-width: 1200px) {
+  .card-columns {
+    column-count: 2;
+  }
+}
+
+/* width */
+::-webkit-scrollbar {
+  width: 10px;
```

Comment:
pls change to em if possible to be consistent

Reply:
Those are the units used for standard breakpoints, I think it should be ok if we use them here

Reply:
Is there not a variable that you could use from bootstrap?

Reply:
Those are only available as classes

Reply:
Yeah I think for things like breakpoints it would be okay considering pixel breakpoints are used in bootstrap.